buildtools/stylelint/stylelint.js

```diff
@@ -0,0 +1,45 @@
+/* eslint no-console: 0 no-undef: 0 */
+const checkStyle = require('stylelint-checkstyle-formatter');
+const fs = require('fs');
+const path = require('path');
+const stylelint = require('stylelint');
+
+const STYLELINT_OUT_FILE = 'build2/OSW-stylelint-checkstyle.xml';
+const SASS_LINT_FILES = [
```

Comment:
looks like there are not much logic and wonder if we could use command line directly?
we could use stylelintignorerc to explicitly config what shall and shall not be lint.

Reply:
+1  Probably not a bad idea to look into.

Reply:
Makes sense! Let me try